apis/extension/constants.go

```diff
@@ -31,4 +31,7 @@ const (
 	LabelPodPriority = DomainPrefix + "priority"
 
 	LabelManagedBy = "app.kubernetes.io/managed-by"
+
+	ReservedByNode     = DomainPrefix + "reserved"
```

Comment:
The style of the newly added Annotation/Label should be consistent with the previous one.
And the name should be `node.koordinator.sh/node-reservation`? Use `NodeDomainPrefix` ?
Perhaps you could define the annotations/labels and corresponding structures into a new file, e.g. named `node_reservation.go`?

Reply:
ok, i will change it later